roles/ufw/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+- name: Install ufw
+  apt:
```

Comment:
This module misses parameters:
1. state, Ansible documentation states:

> Always Mention The State
> The ‘state’ parameter is optional to a lot of modules. Whether ‘state=present’ or ‘state=absent’, it’s always best to leave that parameter in your playbooks to make it clear, especially as some modules support additional states.

2. no version mentioned, this leads to unpredictable provisions and reproducibility problems.

I recommend to add the parameters mentioned above.